### DISCUSSION TITLE
(Question) MAX7219 16x8

### DISCUSSION THREAD

**Discussion by matheusfillipe** [olikraus/u8g2#1371] [Q&A]
I am trying to use two MAX7219, height of 8 and length of 16 (2 displays as length). I haven't found the constructor for it and using the 8x8 constructor makes the same text display on both screens. 

I am doing this:

``` 

/* Constructor */
 U8G2_MAX7219_8X8_F_4W_SW_SPI matrix(U8G2_R0, /* clock=*/ 52, /* data=*/ 51, /* cs=*/ 50, /* dc=*/ U8X8_PIN_NONE, /* reset=*/ U8X8_PIN_NONE);

/* u8g2.begin() is required and will sent the setup/init sequence to the display */
void setup(void) {
  u8g2.begin();
}
/* draw something on the display with the `firstPage()`/`nextPage()` loop*/
void loop(void) {
  u8g2.firstPage();
  do {
    u8g2.setFont(u8g2_font_trixel_square_tn);
    u8g2.drawStr(0,8,"8");
  } while ( u8g2.nextPage() );
  delay(1000);
}
```

I am trying to use both as a single 16x8 display but this constructor isn't
available... I tried looking on how those constructors work but coudl't quite
figure it out yet. It is my first time using displays on arduino.

I tried the 32x8 constructor but then nothing displayed. Would this be possible? How? If you could provide some minimal working code I will figure it out. Thanks!

**Comment by olikraus** [comment]
Unfortunately the MAX7219 deviates a lot from the other display controllers. Adding support is difficult...

**Reply by matheusfillipe** [reply]
Oh i dont mean to add support, just though there was some workaround to make it work at least? there isn't? I was thinking i could like use 32x8 considering it would work on some region and clip for another. Thats not true?

**Reply by olikraus** [reply]
By using the 32x8 device still something should be visible on a 16x8 device. But I don't remember details 🤔
